src/polkadot.scss

```diff
@@ -0,0 +1,79 @@
+// Custom.scss
+// set font family
+$font-family-base: 'Work Sans', sans-serif;
+
+// Button
+$btn-border-radius: 1.5rem;
+$btn-border-radius-lg: 2rem;
+$btn-border-radius-sm: 1.5rem;
+
+.card-button {
+  display: inline-flex !important;
+  align-items: center !important;
+  width: 100% !important;
+  background-color: #FFF !important;
+  border-radius: 8px !important;
+  border-color: #D5DBE0 !important;
+  color: #1E1E1E !important;
+  padding: 0.75rem !important;
+  text-align: left !important; 
+}
+
+.card-button:hover {
```

Comment:
It looks very good. just was thinking we also might want to change the border color to primary-color (for example pink for polkadot!) as an effect.

Reply:
Yeah, I actually also tried the pink border, but found the shadow effect a little less loud. It's also closer to the card hover styling on polkadot.network. I can imagine that for Kusama styling though changing the border color would look really good!